doc/xml/user-guide.xml

```diff
@@ -643,6 +650,23 @@
         <p>Shared access signatures may be used by setting the <br-option>repo1-azure-key-type</br-option> option to <id>sas</id> and the <br-option>repo1-azure-key</br-option> option to the shared access signature token.</p>
     </block-define>
 
+    <!-- ======================================================================================================================= -->
+    <block-define id="gcs-setup">
+        <p><backrest/> supports locating repositories in <proper>GCS-compatible</proper> object stores. The bucket used to store the repository must be created in advance &amp;mdash; <backrest/> will not do it automatically. The repository can be located in the container root (<path>/</path>) but it's usually best to place it in a subpath so object store logs or other data can also be stored in the buket without conflicts.</p>
+
+        <backrest-config host="{[gcs-setup-host]}" file="{[backrest-config-demo]}" owner="{[gcs-setup-config-owner]}">
+            <title>Configure <proper>Azure</proper></title>
```

Comment:
Shouldn't this be `Configure <proper>GCS</proper>` instead of `Configure <proper>Azure</proper>`

Reply:
Fixed in 9c1a0233577bb. Did I proofread this at all?